### PR TITLE
38 support model hash selection in benchit cli

### PR DESCRIPTION
**Pull request by jeremyfowers**
closes 38

Example with no hash targets
```
(mla) jfowers@jfowers:~/mlagility/examples/cli$ benchit two_models.py 

Models found while running 'two_models'

pytorch_model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          SmallModel (<class 'two_models.SmallModel'>)
        Location:       /home/jfowers/mlagility/examples/cli/two_models.py, line 32
        Parameters:     55 (<0.1 MB)
        Hash:           479b1332
        Status:         Model successfully built!

another_pytorch_model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          SmallModel (<class 'two_models.SmallModel'>)
        Location:       /home/jfowers/mlagility/examples/cli/two_models.py, line 40
        Parameters:     510 (<0.1 MB)
        Hash:           215ca1e3
        Status:         Model successfully built!
```

Target the second model with a hash target:
```
benchit two_models.py:215ca1e3 

Models found while running 'two_models'

pytorch_model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          SmallModel (<class 'two_models.SmallModel'>)
        Location:       /home/jfowers/mlagility/examples/cli/two_models.py, line 32
        Parameters:     55 (<0.1 MB)
        Hash:           479b1332

another_pytorch_model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          SmallModel (<class 'two_models.SmallModel'>)
        Location:       /home/jfowers/mlagility/examples/cli/two_models.py, line 40
        Parameters:     510 (<0.1 MB)
        Hash:           215ca1e3
        Status:         Model successfully built!
```

Target both models:
```
Models found while running 'two_models'

pytorch_model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          SmallModel (<class 'two_models.SmallModel'>)
        Location:       /home/jfowers/mlagility/examples/cli/two_models.py, line 32
        Parameters:     55 (<0.1 MB)
        Hash:           479b1332
        Status:         Model successfully built!

another_pytorch_model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          SmallModel (<class 'two_models.SmallModel'>)
        Location:       /home/jfowers/mlagility/examples/cli/two_models.py, line 40
        Parameters:     510 (<0.1 MB)
        Hash:           215ca1e3
        Status:         Model successfully built!
```

### CODE REVIEW

File: src/mlagility/cli.py [hunk 1]
```diff
@@ -355,7 +355,7 @@ def main():
     # we alter argv to insert the command for them.
 
     if len(sys.argv) > 1:
-        if sys.argv[1] not in subparsers.choices.keys() and sys.argv[1].endswith(".py"):
+        if sys.argv[1] not in subparsers.choices.keys() and ".py" in sys.argv[1]:
```
Comment (danielholanda) [inline src/mlagility/cli.py:358]: Isn't this a less robust solution than `endswith`?

**Review by danielholanda — APPROVED** [review]
Looks good. only added a minor comment.

File: src/mlagility/cli.py (hunk 1 above)
Comment (jeremyfowers) [reply src/mlagility/cli.py:358]: Will do something like `name.split(":")[0].endswith(".py")`

Comment (jeremyfowers) [reply src/mlagility/cli.py]: Added a nice decode function that I can call in cli.py and use with endswith()

**Pull request merged**